ear/src/main/resources/s3binaries/nxserver/config/s3directupload-config.xml.nxftl

```diff
@@ -30,13 +30,13 @@
       <property name="region">${nuxeo.s3storage.transient.region}</property>
       <property name="bucket_prefix">${nuxeo.s3storage.transient.bucket_prefix}</property>
       <property name="crypt.serverside">${nuxeo.s3storage.transient.crypt.serverside}</property>
-      <property name="cachesize">100MB</property>
+      <property name="cachesize">${nuxeo.s3storage.cachesize}</property>
```

Comment:
All the changes to this file are not in LTS2021/master, so should be forward-ported as well so that we don't have features in 10.10 but not LTS2021.

Reply:
I'll do it too.